libsql-sqlite3/src/test2.c

```diff
@@ -14,6 +14,8 @@
 ** testing of the SQLite library.
 */
 #include "sqliteInt.h"
+#include "wal.h"
+#include <sys/signal.h>
```

Comment:
I suppose sys/signal.h is some debug residue? You certainly don't need to add posix signals in this code